lte/gateway/docker/services/python/Dockerfile

```diff
@@ -119,6 +119,10 @@ RUN wget -qO - https://artifactory.magmacore.org:443/artifactory/api/gpg/key/pub
 RUN echo "deb https://packages.fluentbit.io/ubuntu/focal focal main" > /etc/apt/sources.list.d/tda.list
 RUN wget -qO - https://packages.fluentbit.io/fluentbit.key | apt-key add -
 
+RUN wget -qO - https://get.docker.com/ > /tmp/get_docker.sh && \
```

Comment:
apt is a better approach to get docker as it supports getting a fixed version

Reply:
Changed it to apt. :+1: